esp/platform/txsummary.hpp

```diff
@@ -78,9 +91,12 @@ class CTxSummary : extends CInterface
     };
 
     using Entries = std::list<Entry>;
+    using Timers = std::map<StringBuffer, CumulativeTimer*>;
```

Comment:
I would have expected something like std::map<StringAttr, Linked<CumulativeTime> >

- StringAttr is much smaller than a StringBuffer
- using a wrapper class means that the mapped objects should be cleaned up correctly